.github/workflows/version_handling.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: Version-handling check
+
+on:
+  pull_request:
+
+jobs:
+  versions:
+    name: Versions
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ['v0.5.0', 'v0.6.0', 'v0.7.0', 'v0.8.0', 'rc', 'latest', 'devel']
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2
+
+      - name: Download and install subctl
+        run: VERSION="${{ matrix.version }}" ./getsubctl.sh
+
+      - name: Check we got subctl
+        run: ~/.local/bin/subctl version
+
+      - name: Check we got the right version (specific)
+        if: ${{ startsWith(matrix.version, 'v') }}
+        run: |
+          [ "$(~/.local/bin/subctl version)" = "subctl version: ${{ matrix.version }}" ]
+
+      - name: Check we got the right version (rc)
+        if: ${{ matrix.version == 'rc' }}
```

Comment:
Shouldn't we also check `devel` in the same manner?

Reply:
Done, and I’ve improved the `rc` check.